internal/cli/runner_profile_inspect.go

```diff
@@ -106,6 +109,9 @@ func (c *RunnerProfileInspectCommand) Run(args []string) int {
 		{
 			Name: "Target Runner ID", Value: targetRunner,
 		},
+		{
+			Name: "Target Runner Labels", Value: targetRunnerLabels,
+		},
```

Comment:
Not a big deal, but a target runner can have only one of any, id, or lables, so should this be one field that is like `Target:` instead that follows the format of `list` or something?

Reply:
Hmm that's a good point actually, right now it's not hard-coded to restrict only one of ID or Labels, but the CLI presents it as such, and ID always takes precedence over labels.